.github/workflows/program-classification-validator.yml

```
@@ -22,21 +22,13 @@ jobs:
   validate-program:
     name: Validate PR Program Classification
 
-    if: |
-      github.event.pull_request.draft == false &&
-      github.actor != 'github-actions[bot]'
+    if: github.event.pull_request.draft == false
 
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout repository
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)